code/game/objects/items/deployable/capsule.dm

```diff
@@ -26,6 +26,12 @@
 	playsound(src, 'sound/machines/button2.ogg', 15, 1)
 	activate_capsule(user)
 
+/obj/item/bluespace_capsule/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	. = ..()
+	if(!.)
+		playsound(src, 'sound/machines/button2.ogg', 15, 1)
+		activate_capsule(null)
+
```

Comment:
This isn't integrated into the main deployable component because BS capsules handles the activation delay on its down (below this)